snooty/specparser.py

```diff
@@ -392,6 +397,16 @@ def resolve_value(key: str, inheritable: _T) -> _T:
         for key, inheritable in inheritable_index.items():
             resolve_value(key, inheritable)
 
+    @classmethod
+    def initialize(cls, text: str) -> None:
+        cls.SPEC = Spec.loads(text)
 
-GLOBAL_SPEC_PATH = util.PACKAGE_ROOT.joinpath("rstspec.toml")
-SPEC = Spec.loads(GLOBAL_SPEC_PATH.read_text(encoding="utf-8"))
+    @classmethod
```

Comment:
Would it make sense to have an error returned if there spec.get results in no spec, or am I being overcautious?

Reply:
Spec.get() could throw TomlDecodeError (if the TOML is invalid), LoadError (if the schema doesn't match what we expect), or some form of OSError if the built-in rstspec can't be read. The assertion should never trip (hence why it's an assertion: we're just proving a property to the type system).

We could conceivably catch these exceptions to print something nicer, but that's more important with external users.